test/onnx/test_fx_op_consistency.py

```diff
@@ -556,7 +490,7 @@
         and sample.kwargs.get("padding") == 1,
         reason="FIXME: After https://github.com/microsoft/onnxruntime/issues/15446 is fixed",
     ),
-    xfail(
+    skip(
```

Comment:
After type promotion this test starts to fail a few subtests with unexpected pass, which appears to be incosistent with the taggex xfail reason.

Reply:
Actually more likely due to the bumping onnxscript version in this PR, so unrelated to type promotion. The inconsistency concern still holds.